.github/workflows/release.yaml

```diff
@@ -103,20 +82,54 @@ jobs:
       - name: Build
         run: |
           make -f builder/Makefile.release
-        # NO tracee:latest (historical reason: amd64 only)
-        # use :aarch64 as tag for the latest image
-        # use :aarch64-v0.xx.0 for versioned image
+        shell: bash
       - name: Publish to docker.io registry
         run: |
           TAG=$(echo ${{ github.event.inputs.ref }} | sed -e "s/v//gI")
           ARCH=$(uname -m)
-          docker image tag tracee:latest aquasec/tracee:${ARCH}
           docker image tag tracee:latest aquasec/tracee:${ARCH}-${TAG}
-          docker image push aquasec/tracee:${ARCH}
           docker image push aquasec/tracee:${ARCH}-${TAG}
+        shell: bash
       - name: Sign Docker image
         run: |
           TAG=$(echo ${{ github.event.inputs.ref }} | sed -e "s/v//gI")
           ARCH=$(uname -m)
-          cosign sign -y $(docker inspect --format='{{index .RepoDigests 0}}' aquasec/tracee:${ARCH})
```

Comment:
Do we have the signing step for the new image?

Reply:
It is not an image, its a manifest. We can sign the manifest as well in a near-future moment, but its 2 different things (the images from the manifest are currently signed).